<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContextPaneWidgetRectangle</class>
 <widget class="QWidget" name="ContextPaneWidgetRectangle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="gradientLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gradient</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QmlEditorWidgets::GradientLine" name="gradientLine" native="true">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QmlEditorWidgets::ColorButton" name="colorColorButton">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="colorSolid">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/qmldesigner/images/icon_color_solid.png</normaloff>:/qmldesigner/images/icon_color_solid.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="colorGradient">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/qmldesigner/images/icon_color_gradient.png</normaloff>:/qmldesigner/images/icon_color_gradient.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="colorNone">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/qmldesigner/images/icon_color_none.png</normaloff>:/qmldesigner/images/icon_color_none.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>9</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Border</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QmlEditorWidgets::ColorButton" name="borderColorButton">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="borderSolid">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/qmldesigner/images/icon_color_solid.png</normaloff>:/qmldesigner/images/icon_color_solid.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="borderNone">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/qmldesigner/images/icon_color_none.png</normaloff>:/qmldesigner/images/icon_color_none.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmlEditorWidgets::ColorButton</class>
   <extends>QToolButton</extends>
   <header location="global">qmleditorwidgets/colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QmlEditorWidgets::GradientLine</class>
   <extends>QWidget</extends>
   <header location="global">qmleditorwidgets/gradientline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
